radixdlt-core/radixdlt/src/jmh/java/org/radix/benchmark/CodecBenchmark.java

```diff
@@ -66,17 +66,15 @@
 
 
 import com.radixdlt.DefaultSerialization;
+import com.radixdlt.serialization.ClassScanningSerializerIds;
+import com.radixdlt.serialization.DeserializeException;
+import com.radixdlt.serialization.DsonOutput.Output;
+import com.radixdlt.serialization.Serialization;
```

Comment:
This seems only an import reordering and same happened to me when I used the _Optimize imports_ option. Do you know how we can fix it?

Reply:
Should we? Some time ago, both Eclipse and Idea were used for code formatting, hence the difference in the import ordering. It is possible to make Eclipse organize imports like Idea does, but not vice versa. Since we're using almost exclusively Idea now, over the time all files will be formatted identically. This file is ancient and was not touched for long time.

Reply:
I can cause many modifications in a file mostly related to import reordering if different people use different conventions.

Reply:
Yes. But we don't commit files without other changes. In this case, changes in imports is irrelevant.